core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/MetricAlreadyExistsException.java

```diff
@@ -21,13 +21,16 @@
  */
 public class MetricAlreadyExistsException extends RuntimeException {
 
-    private final Metric metric;
+    /**  */
```

Comment:
I believe you can get rid of the empty Javadoc comment. Especially for a serial version uid :)